xbmc/pvr/PVRDatabase.cpp

```diff
@@ -449,9 +450,9 @@ bool CPVRDatabase::Delete(const CPVRChannelGroup& group)
   return RemoveChannelsFromGroup(group) && DeleteValues("channelgroups", filter);
 }
 
-bool CPVRDatabase::Get(CPVRChannelGroups& results)
+int CPVRDatabase::Get(CPVRChannelGroups& results)
 {
-  bool bReturn = false;
+  int iLoaded = false;
```

Comment:
even though `false` implicitly coverts to `0` i'd still do `int iLoaded = 0;`

Reply:
Ooops, will change that.